packages/web/server/api/routers/bridge-transfer.ts

```diff
@@ -335,6 +341,24 @@ export const bridgeTransferRouter = createTRPCRouter({
               color: cosmosChain.logoURIs?.theme?.primary_color_hex,
               bech32Prefix: cosmosChain.bech32_prefix,
             } as Extract<BridgeChainWithDisplayInfo, { chainType: "cosmos" }>;
+          } else if (chainType === "bitcoin") {
+            return {
+              prettyName: "Bitcoin",
+              chainId: "bitcoin",
+              chainName: "Bitcoin",
+              chainType,
+              logoUri: "/networks/bitcoin.svg",
+              color: "#F7931A",
+            } as Extract<BridgeChainWithDisplayInfo, { chainType: "bitcoin" }>;
+          } else if (chainType === "solana") {
+            return {
+              prettyName: "Solana",
+              chainId: "solana",
+              chainName: "Solana",
+              chainType,
+              logoUri: "/networks/solana.svg",
+              color: "#9945FF",
```

Comment:
How about we add this static data to our @osmosis-utils package? Similar to Ethereum, I believe it could be useful to share